### ISSUE TITLE
Aborted at 1539174635 (unix time) try "date -d @1539174635" if you are using GNU date ***

### ISSUE THREAD

**Issue by remcova** [facebookresearch/Detectron#701]
Hi guys, when I try to execute the following command:

> python tools/train_net.py \
>     --cfg configs/getting_started/tutorial_1gpu_e2e_faster_rcnn_R-50-FPN.yaml \
>     OUTPUT_DIR /tmp/detectron-output

I get this as output:

> WARNING cnn.py:  25: [====DEPRECATE WARNING====]: you are creating an object from CNNModelHelper class which will be deprecated soon. Please use ModelHelper object with brew module. For more information, please refer to caffe2.ai and python/brew.py, python/brew_test.py for more information.
> WARNING memonger.py:  55: NOTE: Executing memonger to optimize gradient memory
> I1010 12:30:34.254034  3598 memonger.cc:236] Remapping 83 using 19 shared blobs.
> INFO memonger.py:  97: Memonger memory optimization took 0.0129170417786 secs
> I1010 12:30:35.242265  3598 context_gpu.cu:299] GPU 0: 153 MB
> I1010 12:30:35.242468  3598 context_gpu.cu:303] Total: 153 MB
> I1010 12:30:35.298758  3598 context_gpu.cu:299] GPU 0: 311 MB
> I1010 12:30:35.298889  3598 context_gpu.cu:303] Total: 311 MB
> terminate called without an active exception
> *** Aborted at 1539174635 (unix time) try "date -d @1539174635" if you are using GNU date ***
> PC: @     0x7f5bfb446428 gsignal
> *** SIGABRT (@0x3f800000e0e) received by PID 3598 (TID 0x7f5bfc6f3700) from PID 3598; stack trace: ***
>     @     0x7f5bfbefc390 (unknown)
>     @     0x7f5bfb446428 gsignal
>     @     0x7f5bfb44802a abort
>     @     0x7f5bf4e263df __gnu_cxx::__verbose_terminate_handler()
>     @     0x7f5bf4e24b16 __cxxabiv1::__terminate()
>     @     0x7f5bf4e24b4c std::terminate()
>     @     0x7f5bd1b27c90 caffe2::CUDAContext::~CUDAContext()
>     @     0x7f5bd310e59e caffe2::FillerOp<>::~FillerOp()
>     @     0x7f5bd310e647 caffe2::XavierFillOp<>::~XavierFillOp()
>     @     0x7f5be1addbd9 caffe2::SimpleNet::~SimpleNet()
>     @     0x7f5be1b0a916 caffe2::Workspace::RunNetOnce()
>     @     0x7f5be2aa4d38 _ZZN6caffe26python16addGlobalMethodsERN8pybind116moduleEENKUlRKNS1_5bytesEE28_clES6_.isra.3073.constprop.3186
>     @     0x7f5be2aa4ee4 _ZZN8pybind1112cpp_function10initializeIZN6caffe26python16addGlobalMethodsERNS_6moduleEEUlRKNS_5bytesEE28_bJS8_EJNS_4nameENS_5scopeENS_7siblingEEEEvOT_PFT0_DpT1_EDpRKT2_ENUlRNS_6detail13function_callEE1_4_FUNESQ_
>     @     0x7f5be2adc6b0 pybind11::cpp_function::dispatcher()
>     @     0x7f5bfc1e9210 PyEval_EvalFrameEx
>     @     0x7f5bfc1ec4e9 PyEval_EvalCodeEx
>     @     0x7f5bfc1e99b8 PyEval_EvalFrameEx
>     @     0x7f5bfc1ec4e9 PyEval_EvalCodeEx
>     @     0x7f5bfc1e99b8 PyEval_EvalFrameEx
>     @     0x7f5bfc1ec4e9 PyEval_EvalCodeEx
>     @     0x7f5bfc1e99b8 PyEval_EvalFrameEx
>     @     0x7f5bfc1ec4e9 PyEval_EvalCodeEx
>     @     0x7f5bfc1e99b8 PyEval_EvalFrameEx
>     @     0x7f5bfc1ec4e9 PyEval_EvalCodeEx
>     @     0x7f5bfc1e99b8 PyEval_EvalFrameEx
>     @     0x7f5bfc1ec4e9 PyEval_EvalCodeEx
>     @     0x7f5bfc1ec70a PyEval_EvalCode
>     @     0x7f5bfc2059cd run_mod
>     @     0x7f5bfc206b48 PyRun_FileExFlags
>     @     0x7f5bfc207d68 PyRun_SimpleFileExFlags
>     @     0x7f5bfc219f8c Py_Main
>     @     0x7f5bfb431830 __libc_start_main
> Aborted (core dumped)

I'm new to Detectron and i'm struggling for a couple of days now to get it working.... I googled but couldn't find anything that can help me out. Help is really appreciated. See my System information below.

### Expected results
Output.

### Actual results
No output.

### System information
* Operating system: Ubuntu 16.0.4
* Compiler version: 5.4.0
* CUDA version: 9.0.176
* cuDNN version: 7.3.1
* NVIDIA driver version: 410.48 
* GPU: NVIDIA Tesla K80
* `PYTHONPATH` environment variable: /home/username/anaconda3/envs/dopr2/lib/python2.7
* `python --version` output: Python 2.7.15 :: Anaconda, Inc.
* Anything else that seems relevant: I use Anaconda.

**Comment by yiningzeng** [comment]
I have the same problem. Have you solved it

**Comment by remcova** [comment]
Nope, unfortunately I couldn't solve it. I'm currently using RetinaNet with Keras.
